### ISSUE TITLE
Accessibility Audit: Org admin user flows

### ISSUE THREAD

**Issue by mehansen** [CDCgov/prime-simplereport#5773]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the org admin user flows. Do the page scan as well as any relevant guided tests.
Pages to visit:
* `/app/dashboard`
* `/app/settings/*`

Use cases to test:
* view the analytics dashboard with preset and custom date ranges
* add/edit users
* create/edit a facility
* view facilities
* manage org
* view patient self-registration settings

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

**Comment by johanna-skylight** [comment]
Below tickets will track the issues discovered:

https://github.com/CDCgov/prime-simplereport/issues/5814
https://github.com/CDCgov/prime-simplereport/issues/5818
https://github.com/CDCgov/prime-simplereport/issues/5828
